CMakeLists.txt

```
@@ -112,6 +112,12 @@ set(enable-exceptions
   $<$<CXX_COMPILER_ID:GNU>:-fexceptions>
 )
 
+set(enable-fPIC
```

```suggestion
set(enable-fpic
```
For casing consistency with the other flags.